kompose.spec

```diff
@@ -235,20 +234,29 @@ done
 sort -u -o devel.file-list devel.file-list
 %endif
 
+# Why does check need to use buildroot at all? 
 %check
 %if 0%{?with_check} && 0%{?with_unit_test} && 0%{?with_devel}
 %if ! 0%{?with_bundled}
 export GOPATH=%{buildroot}/%{gopath}:%{gopath}
 %else
-export GOPATH=%{buildroot}/%{gopath}:$(pwd)/vendor:%{gopath}
+#export GOPATH=%{buildroot}/%{gopath}:$(pwd)/vendor:%{gopath}
+#export GOPATH=%{buildroot}/%{gopath}:$(pwd):%{gopath}
+export GOPATH=$(pwd):%{gopath}
```

Comment:
Merged https://github.com/gofed/gofed/commit/57bcc3e37fbe3f66c7cc76edd8f176d24597e033

@dustymabe thanks for the fix.